### PR TITLE
Add pr-ci-triggers workflow

### PR DESCRIPTION
**Pull request by AbdYsn**
Added the pr-ci-triggers github workflow to print out available
vendors CIs trigger phrases.

### CODE REVIEW

**Comment by AbdYsn** [comment]
There seem to be a limitation in github where forks are not allowed to read `secrets` and so it is not possible to comment on the PR.

File: .github/workflows/pr-ci-triggers.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+
+jobs:
+  test:
```
Comment (adrianchiris) [inline .github/workflows/pr-ci-triggers.yml:5]: job name : `vendor-ci` ?

File: ci/examples/jenkins/sriov-network-operator-ci.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@
           failure-status: "Build Failed, comment `/test-e2e-all`, /test-e2e-nvidia-all, or `/test-all` to retrigger"
           error-status: "Build Failed, comment `/test-e2e-all`, /test-e2e-nvidia-all, or `/test-all` to retrigger"
           status-context: '{project} CI'
-          trigger-phrase: ".*/test-(all|e2e-all|e2e-nvidia-all(,| |$)).*"
+          trigger-phrase: "^(?!Thanks for your PR).*/test-(all|e2e-all|e2e-nvidia-all(,| |$)).*"
```
Comment (adrianchiris) [inline ci/examples/jenkins/sriov-network-operator-ci.yaml:62]: do you need this change ? the message does not contain the "full" trigger i.e `/<trigger-name>` but rather just `<trigger-name>`

Comment (AbdYsn) [reply ci/examples/jenkins/sriov-network-operator-ci.yaml:62]: Actually, it should contain the full trigger, to make copy pasting it easier

File: .github/workflows/pr-ci-triggers.yml (hunk 1 above)
Comment (AbdYsn) [reply .github/workflows/pr-ci-triggers.yml]: switched it to `vendors-ci-triggers-list`, since i think this would be more descriptive, WDYT?

**Review by adrianchiris — APPROVED** [review]

**Review by e0ne — APPROVED** [review]

**Pull request merged**